### PR TITLE
Prevent DM modal from closing when clicking outside the modal

### LINKED ISSUE #23: Prevent DM from closing when you click outside the DM modal
Sometimes I write a direct message (DM) and accidentally click on the transparent-gray background which closes the DM and loses what I wrote.

### PR DESCRIPTION
**Pull request by filipekiss**
I don't see the need to do anything more complex than this :)

Closes #23

### CODE REVIEW

**Comment by sindresorhus** [comment]
Can you mention it in the feature highlight in the readme?

https://github.com/sindresorhus/refined-twitter#highlights

File: source/content.css [hunk 1]
```diff
@@ -74,3 +74,11 @@ Dashboard
 	float: unset !important;
 	margin: 0 auto !important;
 }
+
+/* Prevent DM Modal from closing when clicking the overlay */
+#dm_dialog {
+	pointer-events: none;
+}
+#dm_dialog-dialog {
+	pointer-events: auto;
```
Comment (sindresorhus) [inline source/content.css:83]: Using `unset` is a better way to reset to the default value.

Comment (filipekiss) [reply source/content.css:83]: Will change, update the README right away.

Comment (sindresorhus) [reply source/content.css:83]: Actually, I was wrong. `unset` doesn't work in this case.

**Comment by sindresorhus** [comment]
Great! :)

**Pull request merged**